x-pack/plugins/security_solution/public/flyout/preview/index.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useMemo } from 'react';
+import { css } from '@emotion/react';
+import type { FlyoutPanel } from '@kbn/expandable-flyout';
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { panels } from './panels';
+
+export type PreviewPanelPaths = 'rule-preview';
+export const PreviewPanelKey: PreviewPanelProps['key'] = 'document-details-preview';
+
+export interface PreviewPanelProps extends FlyoutPanel {
+  key: 'document-details-preview';
+  path?: PreviewPanelPaths[];
+  state?: {
```

Comment:
the more I think about this the more I would prefer not having anything in `state` right now:
- for consistency, until we have a real reason to use `state` I think we should stick to using `params`
- I think the _rule_ we could use to decide when to use `state` instead of `params` could be when we have non-data related information to pass to the panels (I guess we could argue that `title` would be the case here as it is purely UI related and not data...)

Reply:
moved them back to `params`